userguide/directoryservices.rst

```diff
@@ -264,18 +283,15 @@ Click the :guilabel:`REBUILD DIRECTORY SERVICE CACHE` button if a new
 Active Directory user needs immediate access to %brand%. This occurs
 automatically once a day as a cron job.
 
-.. note:: Active Directory places restrictions on which characters are
-   allowed in Domain and NetBIOS names, and limits the length of those
-   names to 15 characters. If there are problems connecting to the
-   realm,
-   `verify
-   <https://support.microsoft.com/en-us/help/909264/naming-conventions-in-active-directory-for-computers-domains-sites-and>`__
-   the settings do not include any disallowed characters. The
-   Administrator account password cannot contain the *$* character. If a
-   *$* exists in the domain administrator password,
-   :command:`kinit` will report a "Password Incorrect" error and
-   :command:`ldap_bind` will report an "Invalid credentials (49)"
-   error.
+Active Directory places restrictions on which characters are allowed in
```

Comment:
This needs to be more specific.

`` Active Directory does not allow :literal:`$` characters in Domain or NetBIOS names. The length of those names is also limited to 15 characters. ``